### ISSUE TITLE
Failed to load resource

### ISSUE THREAD

**Issue by de-salvatierra** [buglloc/brick#55]
Ubuntu 16.04 Белый экран на всё приложение, и сверху справа сообщение Failed to load resource

> ~$ tail ~/.cache/brick/runtime.log
> 
> [1012/151047:INFO:notification_manager_linux.cc(271)] Notification server supports x-canonical-append
> 
> [1012/151048:WARNING:last_resource_provider.cc(16)] Intercepted resource "https://*****.bitrix24.ru//desktop_app/internals/web/pages/portal-loader" was not found.
> 
> [1012/151138:WARNING:pref_notifier_impl.cc(27)] pref observer found at shutdown proxy
> [1012/151140:INFO:notification_manager_linux.cc(271)] Notification server supports x-canonical-append
> 
> [1012/151141:WARNING:last_resource_provider.cc(16)] Intercepted resource "https://*******.bitrix24.ru//desktop_app/internals/web/pages/portal-loader" was not found.

![image](https://cloud.githubusercontent.com/assets/5816848/19309837/ec0093fe-908f-11e6-9346-355b8949e268.png)

**Comment by urichalex** [comment]
> ~$ tail ~/.cache/brick/runtime.log

[1012/151047:INFO:notification_manager_linux.cc(271)] Notification server supports x-canonical-append

[1012/151048:WARNING:last_resource_provider.cc(16)] Intercepted resource "https://*****.bitrix24.ru//desktop_app/internals/web/pages/portal-loader" was not found.

[1012/151138:WARNING:pref_notifier_impl.cc(27)] pref observer found at shutdown proxy
[1012/151140:INFO:notification_manager_linux.cc(271)] Notification server supports x-canonical-append

[1012/151141:WARNING:last_resource_provider.cc(16)] Intercepted resource "https://*******.bitrix24.ru//desktop_app/internals/web/pages/portal-loader" was not found.

**Comment by urichalex** [comment]
![image](https://cloud.githubusercontent.com/assets/5816848/19309761/8a4c8078-908f-11e6-8ce4-8837080e0b88.png)

**Comment by urichalex** [comment]
![image](https://cloud.githubusercontent.com/assets/5816848/19309829/e1f5f5b6-908f-11e6-8f7c-3895f7a9127e.png)

**Comment by buglloc** [comment]
Я, кажется, понял в чем проблема.
Какой указан Host для этого аккаунта? Зайди, пожалуйста, в настройки аккаунта (tray icon -> Accounts -> Manage Accounts -> нужный аккаунт -> Edit) и проверь, что там нет "/" в имени хоста.

Правильно:
![2016-10-12-152056](https://cloud.githubusercontent.com/assets/7323495/19309871/1e55a218-9090-11e6-9a66-1b15a7460702.png)

Не правильно:
![2016-10-12-152037](https://cloud.githubusercontent.com/assets/7323495/19309873/21351b1c-9090-11e6-8528-5de4cc258659.png)

**Comment by buglloc** [comment]
Ну или удали весь `~/.config/brick/`и начни заново:)

**Comment by de-salvatierra** [comment]
Да, это всё моя невнимательность =) Но, думаю, было бы лучше реализовать обрезку слэшей. Так сказать защиту на дурака)

**Comment by buglloc** [comment]
Согласен. К сожалению, в последнее время у меня совсем не осталось времени на этот проект. Поэтому сойдется ли - я не знаю:(
